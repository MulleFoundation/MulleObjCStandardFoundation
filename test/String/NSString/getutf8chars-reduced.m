#import <MulleObjCStandardFoundation/MulleObjCStandardFoundation.h>

#include <stdio.h>
#include <stdlib.h>


int   main( void)
{
   NSString     *obj;
   NSUInteger   len;
   char         buf[ 256];
   char         *params_1[] =
   {
      "L\xc3\xb6\xc3\xb6\xc3\xb6\xc3\xb6orem ipsum dolor sit amet, consectetur "
      "adipisici elit, sed eiusmod "
   };
   NSUInteger  n_1 = sizeof( params_1) / sizeof( params_1[ 0]);

   obj = [[[NSString alloc] initWithUTF8String:params_1[ 0]] autorelease];
   len = [obj mulleGetUTF8Characters:buf
                           maxLength:2];
   mulle_printf( "\"%.*@\": %td %td\n", len, obj, [obj length], len);
   return( 0);
}

