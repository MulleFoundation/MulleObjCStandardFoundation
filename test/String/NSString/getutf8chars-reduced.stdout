"L": 72 1
